pkg/systemd/logsHelpers.js

```diff
@@ -58,6 +58,13 @@
             match.push(...['_SYSTEMD_UNIT=' + s, "+", "COREDUMP_UNIT=" + s, "+", "UNIT=" + s]);
         });
         full_grep += "service:" + options.service + " ";
+    } else if (options["user-service"]) {
+        options["user-service"].split(",").forEach(s => {
+            if (!s.endsWith(".service"))
+                s = s + ".service";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18032-20230116-112945-90f9e191-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)